packages/frontend/src/components/screens/MainScreen/MainScreen.tsx

```diff
@@ -519,55 +495,62 @@ function ChatNavButtons({
   }, [openDialog, chatId])
 
   return (
-    <>
-      <span className='views' data-no-drag-region>
-        {lastUsedApps && lastUsedApps.length > 0 && (
-          <AppIcons accountId={selectedAccountId()} apps={lastUsedApps} />
-        )}
+    <div className='views' data-no-drag-region>
```

Comment:
I'd put `data-no-drag-region` on the actual buttons to make the drag area even more exact, if that is possible.